.github/workflows/main.yml

```diff
@@ -213,11 +208,9 @@ jobs:
 
       - name: Build all
         run: |
-          BUILD=`git rev-parse --short "$GITHUB_SHA"`
-          TAQ_VERSION="${TAQ_VERSION/\//-}"
           npx lerna bootstrap 
           npm run build:packages 
-          deno compile --output taq --allow-run --allow-write --allow-read --allow-env --allow-net --import-map ./import_map.json --no-prompt index.ts --setBuild "$BUILD" --setVersion "$TAQ_VERSION" --lock ./deno-lock.json
+          npm run build:binary
```

Comment:
Since we have to use `./bin/build.sh` in one of the locations, let's turn all `npm run build:binary` into that for consistency sake?